app/.env.example

```diff
@@ -1,5 +1,4 @@
 RELAY_SERVER=<Server>
 PORT=<Port>
 GRAPHQL_API_URL=<URL>
-MAIL_IO_USER=<Username>
-MAIL_IO_PASS=<Password>
+SMTP_CONNECTION_STRING=<connection string>
```

Comment:
We probably need to setup a config map in OpenShift for the `.env` file. @wenzowski can you confirm?

Reply:
Either a ConfigMap or a Secret. Don't we already have a Secret? Checking...

Reply:
(there are password here, so a secret would be better)

Reply:
#423 is adding a secret. BTW the detect-secrets pre-commit hook was being overzealous so I had to disable it there. I guess we can merge this one as-is, preventing it from being deployable, and I can add the secret after #423 is merged

Reply:
Ah, so we have a secret explicitly for postgres but not a secret for app config. Let's add a new secret with some example data just like we do here in `.env.example` and have that secret managed outside the lifecycle of our deployment process (create it with dummy data only if it doesn't exist). We'll still need to bind the secret into our deploymentconfig same as we do for postgres https://github.com/bcgov/cas-ciip-portal/blob/develop/openshift/deploy/deploymentconfig/app.yml#L54